doc/source/analysis.rst

```diff
@@ -1,5 +1,400 @@
 Graph analysis
 ==============
+|igraph| enables analysis of graphs/networks from simple operations such as adding and removing nodes to complex theoretical constructs such as community detection. Read the `API documentation`_ for details on each function and class.
 
-.. note:: TODO. This is a placeholder section; it is not written yet.
+The context for the following examples will be to import the :class:`Graph` class and to have one or more graphs available:
 
+>>> import igraph as ig
```

Comment:
This is just an anecdotal note: it is very interesting to see how the community has agreed on the _de facto_ abbreviation of igraph just like NumPy became `np` and Pandas became `pd` unanimously. (I never import `igraph` on its own, I always use `from igraph import ...` - but I never use `import numpy as np` or `import pandas as pd` either).

Reply:
hehe I'm like Vincent, like my `ig`